SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy
+
+## Supported Versions
+
+Please see [Releases](https://github.com/prysmaticlabs/prysm/releases/). We recommend using the [most recently released version](https://github.com/prysmaticlabs/prysm/releases/latest).
+
+## Reporting a Vulnerability
+
+Please send vulnerability reports to security@prysmaticlabs.com.
```

Comment:
Can we add something here about our security.txt file? This is a signed document outlining the preferred encryption and reporting destination.

https://github.com/prysmaticlabs/prysm/blob/develop/.well-known/security.txt

Reply:
More info on this (draft) standard: https://securitytxt.org/

Reply:
Yeah that's good, updated it! Have you considered putting it under https://prysmaticlabs.com/.well-known/security.txt as well?
You could also consider this if you wish to further increase visibility: https://dnssecuritytxt.org/

Reply:
Great feedback! We could definitely put that on our website. I'll submit a PR for that later this week.

Regarding the dns security txt, i'll update that immediately. Great suggestion!

Reply:
DNS is done!
```
;; QUESTION SECTION:
;prysmaticlabs.com.             IN      TXT

;; ANSWER SECTION:
prysmaticlabs.com.      264     IN      TXT     "google-site-verification=JJw5PVDCCqp9N4HsUSghS63qV1Eyy65mlS4OnGY8s6A"
prysmaticlabs.com.      264     IN      TXT     "v=spf1 include:_spf.google.com ~all"
prysmaticlabs.com.      264     IN      TXT     "security_contact=mailto:security@prysmaticlabs.com"
prysmaticlabs.com.      264     IN      TXT     "security_policy=https://github.com/prysmaticlabs/prysm/blob/develop/.well-known/security.txt"
```